api/scpca_portal/models/sample.py

```diff
@@ -35,8 +35,14 @@ class Modalities:
             SPATIAL: "Spatial Data",
         }
 
+    class AgeTimings(models.TextChoices):
+        COLLECTION = "collection"
+        DIAGNOSIS = "diagnosis"
+        UNKNOWN = "unknown"
```

Comment:
```suggestion
        UNKNOWN = "unknown"
        NA = "NA"
```

We need to account for a potential `NA` value from the input bucket.

Reply:
Are we sure that we want to support an `NA` value? 

In the updates to the docs from PR https://github.com/AlexsLemonade/scpca-docs/pull/331 it only mentions 3 possible values:
- `diagnosis`
- `collection`
- `unknown`

Effectively `unknown` in this case would be acting as a more descriptive `NA`

Reply:
Actually, I think it is safe to remove the choices from this attribute. We should always be populated this based off of the input metadata and the value does not affect anything internally. So we should just keep this as a nullable string.

We need this nullable for the migration and after that we can require it's population at creation time.

Reply:
Actually since we never expect to have a null value during loading we should just have both be a text field with the default as common.NA.

Reply:
I don't think this matters anymore, but `age_timing` can be `NA` if `age` is `NA`.